templates/zerver/help/mastering-the-compose-box.md

```diff
@@ -1,24 +1,72 @@
 # Mastering the compose box
 
-When composing a message, Zulip lets you view a different conversation from the
-one you are composing to. For example, you can start a new topic without
+When composing a message, Zulip lets you start a new topic without
 changing your narrow, send a private message about the topic you're viewing, or
 look up a related discussion.
 
-In this context, the parts of the message view that are outside of the
+In this context, you can view a different conversation from the one you are
+composing to. The parts of the message view that are outside of the
 conversation you are composing to are faded for clarity.
```

Comment:
Hm, I don't think the changes in the intro are an improvement. Can you describe the problem(s) you were trying to solve?

Reply:
I think the intro reads a little better if we first talk about what you can **do** with the compose box, and then what you can  **view** in that context.